config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_runjobs.run.cnrm.cloud.google.com.yaml

```diff
@@ -248,24 +234,24 @@ spec:
                                               - external
                                             properties:
                                               external:
-                                                description: 'Allowed value: The `name`
-                                                  field of a `SecretManagerSecret`
-                                                  resource.'
+                                                description: ' If provided must be
```

Comment:
This introduces an exception on `SecretmanagerSecretRef` normalization. I'd like to address separately

Reply:
Sorry - can you clarify?  This is a description change...

/hold for clarification